.github/workflows/web-app-docker-publish.yml

```
@@ -6,68 +6,68 @@ on:
     branches:
       - master
       - release-*
-    
+
     # Publish `v1.2.3` tags as releases.
     tags:
       - v*
-  
+
   # Run tests for any PRs.
   pull_request:
 
 env:
-  ARCH: linux/ppc64le,linux/amd64,linux/arm64
+  PLATFORMS: linux/ppc64le,linux/amd64,linux/arm64
 
 jobs:
-  # Run tests.
-  test:
-    runs-on: ubuntu-latest
-    
-    steps:
-      - uses: actions/checkout@v2
-      
-      - name: Run tests
-        run: |
-          make docker-build
-  
   # Push image to GitHub Container Registry.
   push:
-    # Ensure test job passes before pushing image.
-    needs: test
-    
     runs-on: ubuntu-latest
+    # Only run on push events, not PRs
     if: github.event_name == 'push'
     permissions:
       packages: write
       contents: read
-    
+
     steps:
-      - uses: actions/checkout@v2
-      
-      - name: Setup QEMU
-        uses: docker/setup-qemu-action@v2
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
         with:
           platforms: ppc64le,arm64
-      
-      - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log into GitHub Container Registry
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      
-      - name: Export version variable
-        run: |
-          # Strip git ref prefix from version
-          TAG=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
-          
-          # Use Docker `latest` tag convention
-          [ "$TAG" == "master" ] && VERSION=latest
-          
-          echo TAG=$TAG >> $GITHUB_ENV
-      
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=schedule
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=sha
+            # Add 'latest' tag for master branch pushes
+            type=raw,value=latest,enable=${{ github.ref == 'refs/heads/master' }}
+
```

multi-architecture

---

Did you mean changing multi-arch to:
```suggestion
- name: Build and push multi-architecture OCI image
```